.github/workflows/gerrit-required-info-yaml-verify.yaml

```diff
@@ -0,0 +1,142 @@
+---
+name: Gerrit Required INFO Yaml Verify
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+    inputs:
+      GERRIT_BRANCH:
```

Comment:
ditto, default params

Reply:
There are _no_ defaults that can be set for these. They are always required to be filled in. The reason being that these are handed in from Gerrit and there is no valid default.